src/main/java/com/spring/core/config/DataInitializer.java

```diff
@@ -0,0 +1,45 @@
+package com.spring.core.config;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.spring.core.dao.impl.InMemoryBasketDaoImpl;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import org.springframework.util.ResourceUtils;
+
+import java.io.File;
+import java.io.IOException;
+
+/**
+ * This class works with json file and gets data from it
+ *
+ * @author Katuranau Maksimilyan
+ */
+@Component
+public class DataInitializer {
```

Comment:
Don't think that it's required functionality according to the task.
Show me, please, where it is in the task?